waf-cloudfront.tf

```diff
@@ -6,7 +6,15 @@ resource "aws_wafv2_web_acl" "waf_cloudfront" {
   scope       = var.scope
 
   default_action {
-    allow {}
+    dynamic "allow" {
+      for_each = var.default_action == "allow" ? [1] : []

```

Comment:
please make sure to pass the variable default_action as allow for any existing usage of cloudfront waf module

Reply:
Hi @jkhan-mydeal  I have introduced a new var (default_action_cloudfront) set to "allow" as default for this specific usecase